src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const confirmedTabId = tabs.some(({ id }) => id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
```

Comment:
What's the purpose of it? You can just find the selected tab using `selectedTabId` and add a default value for it (first tab) using `||` operator